src/main/java/Duke.java

```diff
@@ -1,10 +1,226 @@
+import duke.task.Task;
+import duke.task.Todo;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.command.Command;
+import duke.command.CommandType;
+import duke.exception.DukeException;
+import duke.exception.ExceptionType;
+
+import  java.util.Scanner;
+
 public class Duke {
+
+    public static final String BYE_MESSAGE = "\n\tSee you! Have a nice day!\n";
+    public static final String DEADLINE_IDENTIFIER = "/by";
+    public static final String EVENT_IDENTIFIER = "/on";
+
+    /**
+     * prints a line of 100 character length.
+     *
+     */
+    public static void printLine() {
```

Comment:
Maybe you can leave out the comment since it is not necessary, the method name is self explanatory. In general, if the method name is self-explanatory, it is okay not to write documentation comments for those :)

Reply:
okie! I will look into it! Thanks!